library/elliptics.h

```diff
@@ -655,12 +696,18 @@ int __attribute__((weak)) dnet_process_cmd_raw(struct dnet_net_state *st,
                                                int recursive,
                                                long queue_time,
                                                struct dnet_access_context *context);
+int __attribute__((weak)) n2_process_cmd_raw(struct dnet_net_state *st,
```

Comment:
just curious - why do we use weak symbol here?

Reply:
It is replacer for `dnet_process_cmd_raw`, that has the same attr. I don't know how necessary to have this attr.